api/v1alpha1/nodenetworkconfigurationenactment_types.go

```diff
@@ -31,16 +32,16 @@ type NodeNetworkConfigurationEnactment struct {
 }
 
 func NewEnactment(nodeName string, policy NodeNetworkConfigurationPolicy) NodeNetworkConfigurationEnactment {
+	labels := names.GetRelationshipLabels()
```

Comment:
@qinqon do we want that set on v1alpha1?
i don't think it's braking anything but not sure it's necessary

Reply:
@qinqon can you see this one please ?